components/utils.js

```diff
@@ -87,7 +87,10 @@ export function handleIssueData(data) {
     newIssue.isFeatured = newIssue.labels.includes("Featured Project");
     return newIssue;
   });
-  return dataHandled.sort(sortByUpdatedDate);
+
+  // filter out issues with a status of 'icebox'
+  const filteredData = dataHandled.filter((issue) => issue.status !== "icebox");
+  return filteredData.sort(sortByUpdatedDate);
```

Comment:
@frankhereford this looks like it'll work just fine, but i wonder if we should filter these records from the [Socrata query](https://github.com/cityofaustin/atd-product/blob/46408ab02e0d686481aab71ab16503dc0299712f/components/wrappers/IssuesContextWrapper.js#L7-L8) if we don't need them to come over the wire at all. perhaps ditto for the archived projects as well.

Reply:
This works great here John, thanks for this insightful suggestion. I'm going to leave the archived projects as it's implemented though because that classification is only removed in a certain view (completed) and not across the entire site like this one. Thank you so much for this!